src/components/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,46 @@
+
+import { mount } from 'enzyme';
+import * as React from 'react';
+import Breadcrumbs from './Breadcrumbs';
+
+describe('<Breadcrumbs />', () => {
+
+    describe('Rendering and text', ()=>{
+        it('should render without throwing an error', () => {
+            const cut = mount(<Breadcrumbs />);
+            expect(cut.find('nav > ol.rvt-breadcrumbs')).toHaveLength(1);
+        });
+        it('should take an id', () => {
+            const cut = mount(<Breadcrumbs id="the_id"/>);
+            expect(cut.find('nav#the_id')).toHaveLength(1);
+        });
+        it('should show text children', () => {
+            const cut = mount(<Breadcrumbs>The text</Breadcrumbs>);
+            expect(cut.find('nav > ol.rvt-breadcrumbs > li').text()).toEqual("The text");
+        });
+        it('should wrap multiple children in list items', () => {
+            const cut = mount(
+              <Breadcrumbs>
+                <a href="#">Link One</a>
+                Link Two
+              </Breadcrumbs>);
+            expect(cut.find('nav > ol.rvt-breadcrumbs > li').length).toEqual(2);
+        });
+        it('should mark the last child as the current page', () => {
+            const cut = mount(
+              <Breadcrumbs>
+                <a href="#">Link One</a>
+                Link Two
+              </Breadcrumbs>);
+            expect(cut.find('nav > ol.rvt-breadcrumbs > li').last().prop('aria-label')).toEqual('Current page');
```

Comment:
I believe the current page element should also have `aria-current="page"`

Reply:
I actually think `aria-label="Current page"` is fine

Reply:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.html here is where it mentions aria-current though

Reply:
Thanks for pointing that out.  I was working from an old breadcrumbs example which used `aria-label` but I changed it to `aria-current`.